packages/frontend/src/components/create-campaign/form/rewards-step/reward-tokens-list/index.tsx

```diff
@@ -39,7 +39,19 @@ export function RewardTokensList({
     } = useWatchBalances(address, whitelistedTokens);
 
     return (
-        <div className={styles.root}>
+        <div
+            className={styles.root}
+            style={{
+                height:
+                    whitelistedTokensWithBalance.length === 2
```

Comment:
Not a fan of this approach, can't we have something like:
```tsx
`${whitelistedTokensWithBalance.length * 57}px`
```
(px or rem, but it should be the single row height).